ledger/participant-integration-api/src/main/resources/db/migration/h2database-appendonly/V1__Append_only_schema.sql

```diff
@@ -121,9 +121,18 @@ CREATE TABLE participant_command_completions (
     application_id VARCHAR NOT NULL,
     submitters ARRAY NOT NULL,
     command_id VARCHAR NOT NULL,
+    -- The transaction ID is `NULL` for rejected transactions.
     transaction_id VARCHAR,
-    status_code INTEGER,
-    status_message VARCHAR
+    -- The three columns below are `NULL` if the completion is for an accepted transaction.
+    -- The `rejection_status` contains a Protocol-Buffers-serialized message of type
+    -- `google.rpc.Status`, containing the code, message, and further details (decided by the ledger
+    -- driver).
+    -- The `rejection_status_code` and `rejection_status_message` columns will always be
+    -- `NULL` in an H2-backed index, as data is not persisted across ledger restarts. However, we
```

Comment:
H2 supports disk-based persistence as well, so this statement may be confusing.

Reply:
```suggestion
    -- `NULL` in an H2-backed index, as we do not support disk-based H2 persistence. However, we
```